tracking/protocol.cpp

```diff
@@ -46,17 +111,22 @@ void Protocol::InitHeader(vector<uint8_t> & packet, PacketType type, uint32_t pa
 
   if (!IsBigEndian())
     size = ReverseByteOrder(size);
-  
+
   packet[0] = static_cast<uint8_t>(type);
 }
 
 string DebugPrint(Protocol::PacketType type)
 {
   switch (type)
   {
-    case Protocol::PacketType::AuthV0: return "AuthV0";
-    case Protocol::PacketType::DataV0: return "DataV0";
-    default: return "Unknown";
+  case Protocol::PacketType::AuthV0: return "AuthV0";
+  case Protocol::PacketType::DataV0: return "DataV0";
+  default:
```

Comment:
I guess we don't need default here. Omit default case and return error message out of switch instead.